### PR TITLE
Add support for `apb catalog relist`

### PR DESCRIPTION
**Pull request by dymurray**
Includes small error check for current-context gathering.

Note: I would like to come back to this to use the dynamic go client to grab these resources but this is clean for now.

### CODE REVIEW

**Comment by dymurray** [comment]
Fixes #56

**Comment by djwhatle** [comment]
@dymurray what do I need to do to make this work? I tried this:
```
$ apb catalog relist

**Comment by djwhatle** [comment]
Added more detailed error messages and suggested actions:

__relist attempt without a bearer token__
```
[dwhatley@precision-t apb]$ oc login -u system:admin
Logged into [...] as "system:admin" using existing credentials.
[...]
Using project "default".
[dwhatley@precision-t apb]$ apb catalog relist
ERRO Bearer token not found for current 'oc' user. Log in as a different user and retry. 
INFO View your current token with 'oc whoami -t'  
INFO Some users don't have a token, including 'system:admin' 
```

__relist attempt without privileges to view routes__
```
[dwhatley@precision-t apb]$ apb catalog relist
ERRO Bad relist response. Expected status 200, got: 403
{"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"clusterservicebrokers.servicecatalog.k8s.io \"openshift-automation-service-broker\" is forbidden: User \"apb-user\" cannot get clusterservicebrokers.servicecatalog.k8s.io at the cluster scope: User \"apb-user\" cannot get clusterservicebrokers.servicecatalog.k8s.io at the cluster scope","reason":"Forbidden","details":{"name":"openshift-automation-service-broker","group":"servicecatalog.k8s.io","kind":"clusterservicebrokers"},"code":403}
 
ERRO Current 'oc' user unable to get 'clusterservicebrokers'. Try again with a more privileged user.
 
INFO Administrators can grant 'cluster-admin' privileges with: 
INFO 'oc adm policy add-cluster-role-to-user cluster-admin <oc-user>' 
```

**Comment by djwhatle** [comment]
@dymurray in `catasb` env I found that I was unable to relist without manually specifying the name of my broker, which is expected, but we should put a PR into catasb to fix this (change ansible-service-broker to openshift-ansible-service-broker).

File: cmd/root.go [hunk 1]
```diff
@@ -33,11 +33,11 @@ var CfgFile string
 
 var rootCmd = &cobra.Command{
 	Use:   "apb",
-	Short: "apb is a tool to manage ServiceBundle images",
-	Long: `ServiceBundles are images that represent lifecycle components
-in that they contain all of the orchestration logic to manage
+	Short: "apb is a tool to manage Ansible Playbook Bundle images",
```
Comment (djwhatle) [inline cmd/root.go]: Not really related to your PR, but it feels kind of weird to me that we have a summary of what APBs are in the CLI tool for APBs. I think people expect to see usage instructions when they run the bare command `apb`.

Comment (djwhatle) [reply cmd/root.go]: Meant to add this comment onto the `Long` description line.

**Pull request merged**